starter-code/src/functions-and-arrays.js

```diff
@@ -39,6 +96,22 @@ const wordsUnique = [
   'simple',
   'bring'
 ];
+function uniquifyArray(wordsUnique) {
+  if(wordsUnique.length === 0) return wordsUnique
+  let arrRepe = []
+  for(let i = 0; i< wordsUnique.length; i++){
+    console.log(wordsUnique.indexOf(wordsUnique[i]))
+    if(wordsUnique.indexOf(wordsUnique[i]) !==-1 ){
+      arrRepe.push(wordsUnique[i]);
+      // return arrRepe
+      
+    } else if(wordsUnique.indexOf(wordsUnique[i] === -1)){
+    }
+    // return wordsUnique
+  }
+  return wordsUnique;
+}
+
```

Comment:
Os comento varias cosillas:
- vosotros queréis que cuando las palabras sean únicas se metan en el nuevo array por lo que no tenéis que poner !==-1 , tenéis que poner ==-1 y siempre que sea única se va a ir metiendo
- Lo que queréis es comprobar que no existan en el array que estais creando por lo que no sería  **wordsUnique**.indexOf(wordsUnique[i]) !==-1 tendría que ser **arrRepe**.indexOf(wordsUnique[i]) !==-1
- lo que tenéis que devolver es el nuevo array por lo que el return es :
return **arrRepe**

Por lo que el código que pasa todas las pruebas sería el siguiente:
<img width="552" alt="Captura de pantalla 2020-01-17 a las 11 41 50" src="https://user-images.githubusercontent.com/26923217/72606360-c24f4b00-391e-11ea-9921-36d2c00f0886.png">